vrecord

```diff
@@ -400,10 +401,21 @@ _get_output_filename(){
 
     echo "${DIR}/${PREFIX}${ID}${SUFFIX}.${EXTENSION}"
 }
+
+_get_decklink_inputs(){
+    # get information on what input device options are available
+    #unset DECKLINK_DEVICES
+    if [[ "${DECKLINK_UTILITY_CHOICE}" == "bmdcapture" ]] ; then
+        bmdcapture -h 2>&1 | grep "^->" | head -n 1 | sed 's/.*-> \(.*\) (.*/\1/'
+    else
+        "${FFMPEG_BIN}" -nostdin -v 0 -sources decklink | awk -F'[][]' '{print $2}' | grep -v "^$"
+    fi
+}
 GTKDIALOG_FUNCTION_DOC
 )
 
 # optional dependency checks
+BMDTOOLS_INSTALLED="$(if command -v bmdcapture >/dev/null ; then echo true ; else echo false ; fi)"
```

Comment:
If you choose the name `BMDCAPTURE_INSTALLED`, rather than `BMDTOOLS_INSTALLED `, then it works fine with a function I wrote for the following  six value assignations (PR _in fieri_).